Core/Utilities/Dictionaries.py

```diff
@@ -0,0 +1,14 @@
+""" Collecting utilities for dictionaries
+"""
+
+from itertools import islice
+
+
+def breakDictionaryIntoChunks(aDict, chunkSize):
```

Comment:
That's a nifty piece of code :-) 

Just the docstring is missing

Also you don't need the test on chuncSize, xrange does it for you

Reply:
Done.